### PR TITLE
Pricing page update

### PR DESCRIPTION
**Pull request by fsansalvadore**

### CODE REVIEW

File: apps/www/components/Pricing/ComputePricingCalculator.tsx [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    )
+
+    return price
+  }
+
+  const calculatePrice = () => {
+    let aggregatePrice = 0
+    const computeAggregate = calculateComputeAggregate(aggregatePrice)
+
+    return setActivePrice(computeAggregate + activePlan.price - COMPUTE_CREDITS)
```
Comment (fsansalvadore) [inline apps/www/components/Pricing/ComputePricingCalculator.tsx:76]: @kevcodez can you confirm this compute price calculation is correct?

File: apps/www/components/Pricing/PricingAddons.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    name: 'Custom Domain',
+    heroImg: 'custom-domain-on',
+    icon: 'custom-domain-upgrade',
+    price: 'Flat fee $10/month',
+    description:
+      'Use your own domain for your Supabase project to present a branded experience to your users.',
+    leftCtaText: 'Documentation',
+    leftCtaLink: 'https://supabase.com/docs/guides/platform/custom-domains',
+    rightCtaText: 'See more',
+    rightCtaLink: 'http://',
```
Comment (kevcodez) [inline apps/www/components/Pricing/PricingAddons.tsx:31]: rightCtaLink is unused?

File: apps/www/public/images/pricing/addons-compute-light.png
Comment (kevcodez) [inline apps/www/public/images/pricing/addons-compute-light.png:1]: These files are unused?

File: package-lock.json
Comment (kevcodez) [inline package-lock.json:1]: This bumps framer-motion with 4 major version bumps and it's missing the package.json adjustment?

File: apps/www/components/Pricing/PricingAddons.tsx (hunk 2 above)
Comment (fsansalvadore) [reply apps/www/components/Pricing/PricingAddons.tsx:31]: you're right, sir

File: package-lock.json
Comment (fsansalvadore) [reply package-lock.json:1]: done 👍

File: packages/shared-data/plans.ts [hunk 3]
```diff
@@ -21,106 +22,90 @@ export const plans: PricingInformation[] = [
...
+      'Daily backups stored for 14 days',
       '28-day log retention',
     ],
     featuresPartner: [
-      'Additional Organization member roles',
-      'Daily backups stored for 14 days',
-      'Standardised Security Questionnaire',
       'SOC2',
       'HIPAA available as paid add-on',
+      'Additional member roles',
```
Comment (kevcodez) [inline packages/shared-data/plans.ts:101]:
```suggestion
      'Read only and Billing member roles',
```

File: apps/www/components/Pricing/PricingFAQs.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            target="_blank"
+            href="https://supabase.com/dashboard/support/new"
+            className="transition underline text-brand-link hover:text-brand-600"
+          >
+            Open a support ticket
+          </a>{' '}
+          to receive help from our team of experts.
+        </p>
+        <p className="p text-center">
+          For enterprise enquries,{' '}
```
Comment (kevcodez) [inline apps/www/components/Pricing/PricingFAQs.tsx:49]: enquries

File: apps/www/components/Pricing/PricingFAQs.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+          <a
+            target="_blank"
+            href="https://supabase.com/dashboard/support/new"
```
Comment (kevcodez) [inline apps/www/components/Pricing/PricingFAQs.tsx:52]:
```suggestion
            href="https://forms.supabase.com/enterprise"
```

File: apps/www/components/Pricing/PricingFAQs.tsx (hunk 4 above)
Comment (kevcodez) [inline apps/www/components/Pricing/PricingFAQs.tsx:49]:
```suggestion
          For enterprise enquiries,{' '}
```

File: packages/shared-data/plans.ts [hunk 6]
```diff
@@ -130,20 +115,21 @@ export const plans: PricingInformation[] = [
     href: 'https://forms.supabase.com/enterprise',
     description: 'For large-scale applications managing serious workloads.',
     features: [
-      `Designated Support manager & SLAs`,
-      `On-premise support`,
-      `24×7×365 premium enterprise support`,
-      'Custom Security Questionnaires',
-      `Private Slack channel`,
+      'Designated Support manager',
```
Comment (kevcodez) [inline packages/shared-data/plans.ts:118]: features and featuresPartner should be the same here, so featuresPartner would need to be aligned

**Review by kevcodez — APPROVED** [review]

**Review by alaister — APPROVED** [review]

**Pull request merged**